File: problem_seven.py
# checking small number to large number
a = int(input("Enter the first number:"))
b=  int(input("Enter the second number:"))
c=  int(input("Enter the third number:"))
def smallnumber(a,b,c):
    if  a > b :
       T = a
       a = b
       b = T
    if a > c:
       J = a
       a= c
       c =J
    if  b > c:
       k = b
       b = c
       c =k
    print(a,b,c)

smallnumber(a,b,c)

File: problem_seven.js
// small number to large numbr
var num1=1
var num2 =242
var num3 = 11

function smallNumber(a,b,c){
    if(a>b){
        T = a
        a = b
        b = T
    }
    if(a>c){
        T = a
        a = c
        c = T
    }
    if(b>c){
        T = b
        b = c
        c = T
    }
  console.log(a,b,c)  
}

smallNumber(num1,num2,num3)